docs/examples/tooling/index.md

```diff
@@ -0,0 +1,15 @@
+# Testing Setup
+
+```{toctree}
+:hidden:
+Photodiode Experiment<photodiode>
+Analyzing the Photodiode Experiment <offset>
+```
+
+To ensure our real-time data is accurately timed, we can run a test like a photodiode experiment. This test measures the exact delay between when we tell an event to happen and when it actually does. By using many triggers, we can check for consistency. A script for this type of experiment is provided in the [lsl-tools](https://github.com/WearableSensing/lsl-tools) library on GitHub.
```

Comment:
A "tool" or "package", not quite a script.

Add a bit on how to install/use, and follow this testin setup walkthrough